flake8-rst 0.4.3 config?

With `flake8-rst` 0.4.2 the output of `flake8-rst doc/source --filename="*.rst"` is clean.

With 0.4.3, I see a bunch of failures like

```
doc/source/install.rst:226:14: E999 SyntaxError: invalid syntax
doc/source/sparse.rst:9:4: E402 module level import not at top of file
doc/source/sparse.rst:10:4: E402 module level import not at top of file
doc/source/sparse.rst:10:4: F401 'pandas.util.testing as tm' imported but unused
doc/source/sparse.rst:97:21: E702 multiple statements on one line (semicolon)
doc/source/sparse.rst:247:41: E127 continuation line over-indented for visual indent
doc/source/comparison_with_r.rst:8:4: F401 'numpy as np' imported but unused
doc/source/comparison_with_r.rst:9:31: E225 missing whitespace around operator
doc/source/comparison_with_r.rst:169:6: E121 continuation line under-indented for hanging indent
doc/source/comparison_with_r.rst:169:14: E231 missing whitespace after ','
```

cc @datapythonista will we need some `exclude`s in the `flake8-rst` section of `setup.cfg`?

--- a/ci/deps/travis-36.yaml
+++ b/ci/deps/travis-36.yaml
@@ -9,7 +9,7 @@ dependencies:
   - fastparquet
   - flake8>=3.5
   - flake8-comprehensions
-  - flake8-rst
+  - flake8-rst=0.4.2
   - gcsfs
   - geopandas
   - html5lib

--- a/environment.yml
+++ b/environment.yml
@@ -13,7 +13,7 @@ dependencies:
   - Cython>=0.28.2
   - flake8
   - flake8-comprehensions
-  - flake8-rst
+  - flake8-rst=0.4.2
   - hypothesis>=3.58.0
   - isort
   - moto

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -4,7 +4,7 @@ pytz
 Cython>=0.28.2
 flake8
 flake8-comprehensions
-flake8-rst
+flake8-rst==0.4.2
 hypothesis>=3.58.0
 isort
 moto

